packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu.test.ts

```diff
@@ -623,21 +623,6 @@ describe('pharos-dropdown-menu', () => {
     expect(selected === item).to.be.true;
   });
 
-  it('sets aria attributes on the trigger element', async () => {
```

Comment:
These seems like a decent test, although I wasn't part of these changes, just a check it wasn't removed accidentally

Reply:
Nevermind, I see the code change related to this now below